src/caret_analyze/plot/bokeh/callback_sched.py

```diff
@@ -145,6 +145,7 @@ def sched_plot_cbg(
                    source=rect_source,
                    color=color,
                    alpha=1.0,
+                   legend_label=f'callback{callbcak_num}',
```

Comment:
typo の修正をお願いします。callbcak_num

Reply:
@hsgwa すみません，修正しました。

Reply:
callback_nameという変数名だと、callback.callback_nameと重なってしまうので、変数名はcallback_idxでお願いします。

あと、コメントで以下の内容も追加をお願いできますでしょうか？
「callback_nameをlegendに設定すると、グラフが圧迫されるので、連番を使用する」

Reply:
@bopeng-saitama

Reply:
@hsgwa 
`legend_label=f'callback{callback_idx}',　# Since setting callback name to legend will overwhelm the graph, sequential numbering is used here. `
コメントはこんな感じで書いてもよろしいでしょうか？

Reply:
overwhelmというよりかは、narrowの方が意図に近いです

Reply:
承知いたしました。ありがとうございます。